### PR TITLE
initial raw test

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph 

This is my thinking
- create the same transaction files for couch
- when building the tsung test file we just tell it which folder to use for the transaction XML

Sill to do:
- code to format the template correctly
- how to get the case_json (should be able to re-use code in `loaders.py`)
- how to do case indexes - maybe we shouldn't worry about this use case since it make things much more complicated

### CODE REVIEW

**Comment by benrudolph** [comment]
that makes sense to me. will pull the branch and test it out

**Comment by benrudolph** [comment]
oy just saw your commits, i think we did a bit of duplication. i'm gonna go ahead and say we go with yours since it's cleaner

File: tsung/raw_transactions/couch/get_cases_by_owner_modified.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<transaction name="get_cases_by_owner_modified">
+    <request subst="true">
+        <match do="dump" when="match">total_rows</match>
+        <match do="dump" when="nomatch">total_rows</match>
+        <http method="GET" url="http://{{ host }}:{{ port }}/{{ couch_case_db }}/_design/cases/_view/by_owner_server_modified_on?startkey%3D%5B%22%%_domain%%%22%2C%22%%_user_id%%%22%2C%22%%_modified_since%%%22%5D%26endkey%3D%5B%22%%_domain%%%22%2C%22%%_user_id%%%22%2C%7B%7D%5D"></http>
```
Comment (benrudolph) [inline tsung/raw_transactions/couch/get_cases_by_owner_modified.xml]: doesn't quite work yet, having a hard time getting tsung to parse the url parameters

File: tsung/raw_transactions/couch/get_related_cases.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<transaction name="get_related_cases">
+    <request subst="true">
+        <match do="dump" when="match">total_rows</match>
+        <match do="dump" when="nomatch">total_rows</match>
+        <http method="GET" url="http://{{ host }}:{{ port }}/{{ couch_case_db }}/_design/cases/_view/related?startkey%3D%5B%22%%_domain%%%22%2C%22%%_case_id%%%22%2C%5D%26endkey%3D%5B%22%%_domain%%%22%7B%7D%5D"></http>
```
Comment (benrudolph) [inline tsung/raw_transactions/couch/get_related_cases.xml]: same with other view

File: tsung/raw_transactions/couch/new_case.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
...&quot;, &quot;owner_id&quot;: &quot;%%_user_id%%&quot;, &quot;closed_by&quot;: null, &quot;closed_on&quot;: null, &quot;computed_&quot;: {}, &quot;full_name&quot;: &quot;Silent Bob&quot;, &quot;health_id&quot;: &quot;12345&quot;, &quot;location_&quot;: [], &quot;opened_by&quot;: &quot;320fc8a02431d2e619e261ff0c67d8e9&quot;, &quot;opened_on&quot;: &quot;2013-07-16T12:53:34Z&quot;, &quot;referrals&quot;: [], &quot;xform_ids&quot;: [&quot;6b2891b1-a692-4e82-91b5-a7a843502926&quot;, &quot;d223c854-b731-45cb-ad36-e5aa12d71e36&quot;], &quot;export_tag&quot;: [], &quot;first_name&quot;: &quot;Silent&quot;, &quot;#export_tag&quot;: [&quot;domain&quot;, &quot;type&quot;], &quot;external_id&quot;: null, &quot;modified_on&quot;: &quot;2015-08-29T11:15:38.458106Z&quot;, &quot;delivery_place&quot;: &quot;home&quot;, &quot;weight_at_birth&quot;: &quot;&quot;, &quot;curr_danger_type&quot;: &quot; &quot;, &quot;curr_referral_type&quot;: &quot;&quot;, &quot;server_modified_on&quot;: &quot;%%_utcnow%%&quot;, &quot;stop_postnatal_sms&quot;: &quot;1&quot;, &quot;delivery_in_facility&quot;: &quot;no&quot;, &quot;computed_modified_on_&quot;: null, &quot;initial_processing_complete&quot;: false}"></http>
```
Comment (benrudolph) [inline tsung/raw_transactions/couch/new_case.xml]: ugly, but works. could potentially use `contents_from_file`

**Comment by benrudolph** [comment]
got couch almost working locally. thought i'd push it in case you're going to look at it

**Comment by snopoke** [comment]
@benrudolph thanks, got this working locally and without any errors in pg or couch. Think its GTG.

**Comment by snopoke** [comment]
@benrudolph I'm going to wait to merge this until after the receiver benchmarks are done in case it breaks something.

**Pull request merged**